graphile.config.mjs

```diff
@@ -5,10 +5,30 @@ import { makeV4Preset } from "postgraphile/presets/v4";
 import { PostGraphileConnectionFilterPreset } from "postgraphile-plugin-connection-filter";
 import { PgAggregatesPreset } from "@graphile/pg-aggregates";
 import { PgManyToManyPreset } from "@graphile-contrib/pg-many-to-many";
+import { makeWrapPlansPlugin } from "graphile-utils";
+import { lambda } from "postgraphile/grafast";
 // import { PgSimplifyInflectionPreset } from "@graphile/simplify-inflection";
 
 // For configuration file details, see: https://postgraphile.org/postgraphile/next/config
 
+const BrokenExample = makeWrapPlansPlugin({
+  Mutation: {
+    createTest(plan, _$source, fieldArgs) {
+      const $test = fieldArgs.get(["input", "test" ]);
+
+      const $payload = plan();
+      const $insert = $payload.get("result");
+      const $testDep = lambda(
+        $test,
+        (test) => ({"test": "bad"}),
+        true,
+      );
+      $insert.set("dep", $testDep);
```

Comment:
The issue was that this needs `return $payload;` at the end. Misleading error bug filed here: https://github.com/graphile/crystal/issues/1864